news/698.update.rst

```diff
@@ -0,0 +1 @@
+Update the hook for sklearn.neighbors to account for removed hidden import neighbors._typedef
```

Comment:
Please put the module names into double backtick-quotes so that they are properly styled.